provider/go.mod

```diff
@@ -5,8 +5,8 @@ go 1.19
 require (
 	github.com/hashicorp/go-azure-sdk v0.20230511.1094507
 	github.com/hashicorp/terraform-provider-azuread/shim v0.0.0
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.60.1
```

Comment:
Changes:

https://github.com/pulumi/pulumi-terraform-bridge/compare/v3.60.1...e3aabd38b8f364a8df8522f334ad787b06e33a42

Should probably upgrade bridge to latest released version first.